test/commands/forward.js

```diff
@@ -0,0 +1,37 @@
+var path = require('path');
+var child = require('child_process');
+
+function remove(arg) {
+  var port = arg.split(':')[1];
+  var command = process.platform === 'win32' ?
+    `FOR /F "tokens=4 delims= " %P IN ('netstat -a -n -o ^| findstr :${port}') DO TaskKill.exe /PID %P` :
```

Comment:
LOL yeah, if you want to nuke future Windows support I'd probably be OK with that since there are other things that still don't work with Windows.